### PR TITLE
Fix. Firefox dark and light themes support

### PR DESCRIPTION
**Pull request by hdwilber**
This fix is trying to solve this issue #130
It has been added SVG images to support dark and light themes in firefox.
Also, updated a favicon logo, however, not sure about this.

## New favicon
new favicon

## Dark theme icon support in FF
dark theme support in FF

### CODE REVIEW

File: firefox/manifest.json [hunk 1]
```diff
@@ -11,21 +11,19 @@
     }
   },
   "icons": {
-    "16": "dist/img/favicon/favicon-16x16.png",
-    "32": "dist/img/favicon/favicon-32x32.png",
-    "64": "dist/img/favicon/favicon-64x64.png",
-    "96": "dist/img/favicon/favicon-96x96.png",
-    "128": "dist/img/favicon/favicon-128x128.png"
+    "32": "dist/img/rested-logo.svg"
```
Comment (esphen) [inline firefox/manifest.json]: I had to look this up. It turns out that even though you use SVG and only one file, you still should provide all the different sizes. Could you please provide them like before here, and in 16 and 32 in `default_icon` and `theme_icons`?

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/icons#SVG

File: firefox/manifest.json [hunk 2]
```diff
@@ -11,21 +11,19 @@
...
   "browser_action": {
     "default_title": "Open RESTED tab",
-    "default_icon": {
-      "16": "dist/img/favicon/favicon-16x16.png",
-      "32": "dist/img/favicon/favicon-32x32.png",
-      "64": "dist/img/favicon/favicon-64x64.png",
-      "96": "dist/img/favicon/favicon-96x96.png",
-      "128": "dist/img/favicon/favicon-128x128.png"
-    }
+    "browser_style": true,
```
Comment (esphen) [inline firefox/manifest.json]: Any reason you would need to add `browser_style`? I don't see you using any `.browser-style` classes here.

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface/Browser_styles

Comment (hdwilber) [reply firefox/manifest.json]: Sorry. My mistake. I was testing some other things in order to display the right icon color.

**Comment by hdwilber** [comment]
@esphen .. why my changes were not updated, here? Oh gosh... I thought that I had pushed the changes requested almost 11 days ago :D ... 
I am going to push them, tonight!

**Comment by hdwilber** [comment]
@twbarber... 
Looks like @esphen is really busy. He never saw my updates, again :rofl: 
I can merge it, but I do prefer to get an approval before to do so. Sorry

**Comment by esphen** [comment]
@hdwilber is unfortunately right, I don't have much time for activities these days :expressionless: Feel free to merge PRs you see fit, though, I trust you :)

Anyway, I just had a look through the PR and it looks good to me. So I just merged it, and I'll cut a release later this week. If you guys want anything else in this release, feel free to open PRs before then.

**Pull request merged**